v0/destinations/marketo_static_list/transform.js

```diff
@@ -115,7 +143,7 @@ const process = async event => {
       DESTINATION
     );
   }
-  const response = processEvent(event.message, event.destination, token);
+  const response = processEvent({ ...event, token });
```

Comment:
Why are we spreading the event ? And during de-structuring we are using spread operator again.
I don't see any reason for spread operator here
please check: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
I see, internally when de-structuring
```
 const { token, ...event } = input;
 const { message, destination } = event;
 ``` 
 
 This becomes but confusing can we do something like this
 ```
 const { token, message, destination } = input;
 ```

Reply:
Yes we can do that.